heron/common/src/java/com/twitter/heron/common/basics/TypeUtils.java

```diff
@@ -87,6 +87,17 @@ public static ByteAmount getByteAmount(Object o) {
     }
   }
 
+  public static ByteAmount getByteAmountMB(Object o) {
```

Comment:
@billonahill - It seems we don't have `ByteAmountUnit` class currently. Do you think it is a good idea if we add a new class in `c.t.h.common.basics` in this PR?